notebooks/02-ingestion/14-native-filters.ipynb

```diff
@@ -0,0 +1,815 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "0cb3b009-ebde-4d56-9d59-a028d66d8309",
+   "metadata": {},
+   "source": [
+    "# Filtering incoming stream data using native functions\n",
+    "<!--\n",
+    "  ~ Licensed to the Apache Software Foundation (ASF) under one\n",
+    "  ~ or more contributor license agreements.  See the NOTICE file\n",
+    "  ~ distributed with this work for additional information\n",
+    "  ~ regarding copyright ownership.  The ASF licenses this file\n",
+    "  ~ to you under the Apache License, Version 2.0 (the\n",
+    "  ~ \"License\"); you may not use this file except in compliance\n",
+    "  ~ with the License.  You may obtain a copy of the License at\n",
+    "  ~\n",
+    "  ~   http://www.apache.org/licenses/LICENSE-2.0\n",
+    "  ~\n",
+    "  ~ Unless required by applicable law or agreed to in writing,\n",
+    "  ~ software distributed under the License is distributed on an\n",
+    "  ~ \"AS IS\" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY\n",
+    "  ~ KIND, either express or implied.  See the License for the\n",
+    "  ~ specific language governing permissions and limitations\n",
+    "  ~ under the License.\n",
+    "  -->\n",
+    "\n",
+    "During streaming ingestion, filters can be applied to incoming data by using native filters inside the `transformSpec`. This tutorial demonstrates how to apply some filters as [filters](https://druid.apache.org/docs/latest/querying/filters) against a stream of events.\n",
```

Comment:
```suggestion
    "During streaming ingestion, you can filter incoming data using Apache Druid native filters within the ingestion spec. This tutorial demonstrates how to apply native [filters](https://druid.apache.org/docs/latest/querying/filters) to a stream of events.\n",
```

I don't think you need to mention `transformSpec` in the introduction. Maybe say "ingestion spec" to keep it general?